packages/core/saved-objects/core-saved-objects-migration-server-internal/src/document_migrator/document_migrator.ts

```diff
@@ -149,10 +154,11 @@ export class DocumentMigrator implements VersionedTransformer {
    */
   public migrate(
     doc: SavedObjectUnsanitizedDoc,
-    { allowDowngrade = false }: DocumentMigrateOptions = {}
+    { allowDowngrade = false, targetTypeVersion }: DocumentMigrateOptions = {}
```

Comment:
`targetTypeVersion` was implemented (and tested) at the pipeline level, but wasn't exposed as option at the migrator's level because we did not need it. The changes are quite simple.